.github/workflows/playwright.yml

```diff
@@ -38,15 +35,17 @@ jobs:
       - name: Grab latest dictionaries from dictionaries branch
         uses: actions/checkout@v3
         with:
+          repository: themoeway/yomitan # so that this works on forks
           ref: dictionaries
           path: dictionaries
 
       - name: Grab latest screenshots from master branch
         uses: dawidd6/action-download-artifact@5e780fc7bbd0cac69fc73271ed86edf5dcb72d67 # pin@v2
         continue-on-error: true
-        id: download-screenshots
+        id: master-screenshots
```

Comment:
I notice we are still using `master` eh. Maybe we want to change everything to `main` at some point

Reply:
Yeah might as well, will do it at some point.